test/acceptance/kafo_configure_test.rb

```diff
@@ -340,5 +340,66 @@ module Kafo
         _(stdout).must_include "Hello Kafo\nGoodbye"
       end
     end
+
+    describe 'with classes' do
+      it 'includes enable flag by default' do
+        code, out, err = run_command '../bin/kafo-configure --help'
+        _(code).must_equal 0, err
+        _(out).must_include "--[no-]enable-testing"
+      end
+
+      it 'does not show disable flag if class cannot be disabled' do
+        config = YAML.load_file(KAFO_CONFIG)
+        config[:classes] = {:testing => {:can_disable => false}}
+        File.open(KAFO_CONFIG, 'w') do |file|
+          file.write(config.to_yaml)
+        end
+
+        code, out, err = run_command '../bin/kafo-configure --help'
+        _(code).must_equal 0, err
+        _(out).wont_include "--[no-]enable-testing"
+      end
+
+      it 'shows enable flag if class is disabled' do
+        config = YAML.load_file(KAFO_CONFIG)
+        config[:classes] = {:testing => {:can_disable => false}}
+        File.open(KAFO_CONFIG, 'w') do |file|
+          file.write(config.to_yaml)
+        end
+
+        answers = {'testing' => false}
+        File.open(KAFO_ANSWERS, 'w') do |file|
+          file.write(answers.to_yaml)
+        end
+
+        code, out, err = run_command '../bin/kafo-configure --help'
+        _(code).must_equal 0, err
+        _(out).must_include "--enable-testing"
+      end
```

Comment:
does that mean that when I've run `that-installer --enable-testing` once, the next run will error out because `--enable-testing` is not an option anymore? I can see some (a|se)nsible users cry about that.

Reply:
That is a fair point. My thought was, if the option does nothing, why include it. There are a couple ideas:

 1) If a module is already enabled and cannot be disabled, promote it to `--full-help` to get it out of the users way but still exist
 2) I have been thinking of a better option for ansible-style users, where users can provide a file as input that maps to installer parameters. That would be a nicer interface for those type of users. Though it may suffer from the same problem (no such option).

Reply:
Demoting it to `full-help` sounds good to me.

While we can (and should) make the Ansible life easier by making it *understand* the installer, we also have users out there who have a 50+ page PDF "how to install/upgrade $project" and they will re-use the same installer parameters on every installer invocation.

Is that (repetition of the params on every invocation) needed? Of course not.
Will the users be totally confused that they have feature X enabled, and yet the installer tells them there is no such thing as feature X (which is how they will read the absence of `--enable-X`)? Absolutely.
(As me how I know.)